scripts/test_mail-alarm.py

```diff
@@ -53,7 +53,7 @@ def setUp(self):
         try:
             self.work_dir = tempfile.mkdtemp(prefix='test-mail-alarm-')
             log_file_global = os.path.join(self.work_dir, 'user.log')
-            src_file = './scripts/mail-alarm'
+            src_file = './mail-alarm'
```

Comment:
Can you give an example why this was needed? Are there other tests that rely on this script and invoke it in a different way, or was this a purely manual test before?

Reply:
Tests are failing because of this change, please check the file `.travis-python-nosetests-3.sh` to see how the tests are run. For local testing I recommend a python virtual environment with `mock` and `six` installed.